crates/interpreter/tests/spec.rs

```diff
@@ -164,6 +164,9 @@ impl<'m> Env<'m> {
     }
 
     fn maybe_instantiate(&mut self, name: &str, wasm: &[u8]) -> Result<InstId, Error> {
+        let side_table = prepare(wasm)?;
+        let merged = merge(wasm, side_table)?;
```

Comment:
nit: from UX perspective, we should probably only expose a `prepare(binary: &[u8]) -> Result<Vec<u8>>` function that directly returns the new module.

Reply:
Yes. But this can be done in a separate PR. That said I'm going to review this one when I'm back to the office next week. So you may want to do it in this PR already and move the PR to the main repo (vs your fork).